packages/web-twig/src/Resources/components/Pagination/Pagination.stories.twig

```diff
@@ -0,0 +1,17 @@
+{% extends 'layout/plain.html.twig' %}
+
+{% block content %}
+
+{# Pagination Current First #}
+{% include '@components/Pagination/stories/PaginationCurrentFirst.twig' %}
```

Comment:
Maybe the stories could have headlines like in [the React implementation](https://deploy-preview-879--spirit-design-system-react.netlify.app/src/components/pagination/)?

![obrazek](https://github.com/lmc-eu/spirit-design-system/assets/5614085/84f4e714-1319-4a2e-b648-76e3188fcc34)

Reply:
Fixed that.